dev/io.openliberty.jakarta.websocket.2.0/bnd.bnd

```diff
@@ -19,12 +18,12 @@ Export-Package: \
 	jakarta.websocket.server; version="2.0.0"
 
 Include-Resource: \
-  @${repo;jakarta.websocket:jakarta.websocket-api;2.0.0}!/META-INF/LICENSE, \
-  @${repo;jakarta.websocket:jakarta.websocket-api;2.0.0}!/META-INF/NOTICE
+  @${repo;jakarta.websocket:jakarta.websocket-api;[2.0.0,2.1.0)}!/META-INF/LICENSE, \
+  @${repo;jakarta.websocket:jakarta.websocket-api;[2.0.0,2.1.0)}!/META-INF/NOTICE
 
 instrument.disabled: true
 
 publish.wlp.jar.suffix: dev/api/spec
 
 -buildpath: \
-	jakarta.websocket:jakarta.websocket-api;version=2.0.0
+	jakarta.websocket:jakarta.websocket-api;version="[2.0.0,2.1.0)"
```

Comment:
Whats the reasoning for the version range?  Couldn't it be `strategy=exact` instead?

Reply:
Exact might be the better option? If the version of the rebundled API changes even by a small increment then there is a chance that the exported package versions will also need to be changed. Not many of these Jakarta projects use semantic versioning so the package versions tend to be the same as the API version.

Reply:
This isn't semantic versioning.  This is the GAV on where it is in the repository.  I got this from you actually in what you have done in your bnds and started doing the same.  Sounds like most people would prefer exact and require update the bnd in two places and the oss maven file as well whenever making a change.  I was trying to remove one file needing updates when moving to a new version of the bundle in maven / artifactory.

Reply:
Using ranges in the CDI bundles seemed like it would save us some time but really it doesn't; we still need to check the exported package versions each time since they don't match the overall API version.

Reply:
I have updated to use EXACT everywhere except from the existing usage in CDI and persistence (JPA).